ondemand.osc.edu/central_config.yaml

```diff
@@ -0,0 +1,261 @@
+# Purpose: The email address for your site's user support staff
+# Type: String | null
+# Default:
+OOD_DASHBOARD_SUPPORT_EMAIL: oschelp@osc.edu
+## OOD_DASHBOARD_TITLE
+# Purpose: The title for your OnDemand instance; displayed on the Dashboard
+# Type: String | null
+# Default: Open OnDemand
+OOD_DASHBOARD_TITLE: OSC OnDemand
+## OOD_DATAROOT
+# TODO: revisit this
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_DATAROOT: null
+## OOD_DEV_APPS_ROOT
+# Purpose: Contains the path to the user-specific development directory
+# TODO: What sets this? Do we ever want to override it? Is it always user-specific?
+# Type: String | null
+# Default: /dev/null
+OOD_DEV_APPS_ROOT: null
+## OOD_DEV_SSH_HOST
+# Purpose: A web node, serving OnDemand, where developers can SSH to perform development work.
+# Type: String | null
+# Default:
+OOD_DEV_SSH_HOST: ondemand-test.osc.edu
+## OOD_FILE_EDITOR
+# Purpose: A web-servable path to an alternative file editor application
+# Type: String | null
+# Default: /pun/sys/file-editor/edit
+OOD_FILE_EDITOR: null
+## OOD_LOAD_EXTERNAL_BC_CONFIG
+# Purpose: Flags whether to load external configuration for Batch Connect apps; impacts sub apps
+# TODO: Help me understand sub apps
+# Type: Boolean | null
+# Default: true
+OOD_LOAD_EXTERNAL_BC_CONFIG: false
+## OOD_LOAD_EXTERNAL_CONFIG
+# Purpose: Flags whether to load external configuration; includes initializers, and custom views
+# Type: Boolean | null
+# Default: true
+OOD_LOAD_EXTERNAL_CONFIG: false
+## OOD_LOCALE
+# Purpose: Sets the locale/translation to use
+# Type: String | null
+# Default: en
+OOD_LOCALE: null
+## OOD_LOCALES_ROOT
+# Purpose: File system path where localizations are stored
+# Type: String | null
+# Default: /etc/ood/config/locales
+OOD_LOCALES_ROOT: null
+## OOD_NAVBAR_TYPE
+# Purpose: The Boostrap navbar specialization
+# Type: String | null
+# Default: inverse
+OOD_NAVBAR_TYPE: null
+## OOD_PORTAL
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_PORTAL: null
+## OOD_QUOTA_PATH
+# Purpose: The list of files used to specify the user's file system usage
+# Type: Sequence<String> | null
+# Default: null
+OOD_QUOTA_PATH:
+  - /users/reporting/storage/quota/netapp.netapp-home.ten.osc.edu-users_quota.json
+  - /users/reporting/storage/quota/gpfs.project_quota.json
+  - /users/reporting/storage/quota/gpfs.scratch_quota.json
+## OOD_SHELL
+# Purpose: A path to the shell app with an optional host to connect to
+# Type: String | null
+# Default: /pun/sys/shell/ssh/default (where default is DEFAULT_SSHHOST)
+OOD_SHELL: null
+## OOD_SHELL_URL
+# Purpose: A web-servable path to an alternative terminal emulator
+# Type: String | null
+# Default: /pun/sys/shell
+OOD_SHELL_URL: null
+## OOD_SSH_HOSTS
+# Purpose: ???
+# Type: Sequence<Mapping<>> | null
+# Example:
+# - ssh_host: owens.osc.edu
+#   host_name: Owens
+# - ssh_host: ruby.osc.edu
+#   host_name: Ruby
+# - ssh_host: pitzer.osc.edu
+#   host_name: Pitzer
+# Default: null
+OOD_SSH_HOSTS: null
```

Comment:
OOD_SSH_HOSTS: The shell app should read the cluster config, and this option should be deprecated.

Reply:
Note: its possible this would require the server side portion of the shell app to be rewritten in Ruby, which should be possible now.